src/Controller/ProgrammeController.php

```diff
@@ -69,4 +75,34 @@ public function showExactSearchByName(Request $request): Response
 
         return new JsonResponse($serializedData, Response::HTTP_OK, [], true);
     }
+
+    /**
+     * @Route(path="/sort/participants", methods={"GET"})
+     */
+    public function showSortedByCertainField(Request $request): Response
+    {
+        $sortType = $request->query->get('sort');
```

Comment:
Not providing the sort parameter would make the $sortType variable null and the script will fail on tine 88 since it expects a string for the second argument.

Reply:
Fixed, thanks :)